app/views/documents/history/_whitehall_entry.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="app-timeline-entry">
+<div class="app-timeline-entry app-timeline-entry--highlighted">
```

Comment:
It looks like this will highlight every entry from Whitehall whereas we only want to highlight a few of them (internal note, fact checks). 

It's worth looking at the approach we used in https://github.com/alphagov/content-publisher/blob/b3b61fe7514ff7362192e34c0403edc1237bd134/app/views/documents/history/_content_publisher_entry.html.erb#L40 to determine whether something gets a highlight - we should probably have this covered with a unit test too.

We have a story to make them into a shared component for later to deal with the duplicated logic: https://trello.com/c/iadUckIy/1429-convert-the-timeline-entry-partials-into-a-component

Reply:
@kevindew  I read [this](https://github.com/alphagov/content-publisher/pull/1643#discussion_r367391070) and probably misunderstood Simon's intent.  I've updated as per your suggestion and will write a test as part of @emmabeynon's [review](https://github.com/alphagov/content-publisher/pull/1723#discussion_r374180783).

Personally, I prefer the approach you put forward [here](https://github.com/alphagov/content-publisher/pull/1643#discussion_r366489776) - it feels much more flexible.